test/join.jl

```diff
@@ -578,4 +578,23 @@ module TestJoin
         @test eltypes(join(l, r, on=:b, kind=:outer, makeunique=true)) ==
             [Union{Int, Missing}, CS, Union{Int, Missing}]
     end
+
+    @testset "test checks of merge key uniqueness" begin
+
```

Comment:
No need for all these empty lines here nor below (see rest of the file for examples).